src/WaveTrack.cpp

```diff
@@ -983,18 +1138,19 @@ void WaveTrack::ClearAndPaste(double t0, // Start of time to clear
             double st;
             double et;
 
-            st = clip->GetStartTime();
-            et = clip->GetEndTime();
+            st = clip->GetPlayStartTime();
+            et = clip->GetPlayEndTime();
```

Comment:
I think cut line operations should only refer to sequence start and end times.  The mOffset of a cut line was always interpreted as relative to the mOffset of the containing clip.

I think there is not a real bug because the trim left value of a cut line never becomes nonzero, but then it is simpler not to rely on that.